coalib/bears/Bear.py

```diff
@@ -336,14 +336,33 @@ def check_prerequisites(cls):
         """
         Checks whether needed runtime prerequisites of the bear are satisfied.
 
-        This function gets executed at construction and returns True by
-        default.
+        This function gets executed at construction and checks by default all
+        prerequisites specified in the ``REQUIREMENTS`` field. When no
+        ``REQUIREMENTS`` are specified, this function returns True.
 
         Section value requirements shall be checked inside the ``run`` method.
 
         :return: True if prerequisites are satisfied, else False or a string
                  that serves a more detailed description of what's missing.
         """
+        for requirement in cls.REQUIREMENTS:
+            if not requirement.check():
+                error_string = (
+                    "Requirement for '{}' ({}) is not satisfied.".format(
+                        requirement.package, requirement.version))
+
+                if isinstance(requirement, PackageRequirement):
+                    try:
+                        # FIXME What about shell-quotation for returned
+                        # FIXME `install_command`?
+                        error_string += (
+                            '\nYou can install it using this command:\n' +
+                            ' '.join(requirement.install_command))
```

Comment:
uhm all requirements should have an install command, this shouldn't be specific to the package requirement